### PR TITLE
[14.0][FIX] shopfloor_checkout_package_measure: validate on done click

### PR DESCRIPTION
**Pull request by TDu**
ref.: rau-135

### CODE REVIEW

File: shopfloor_checkout_package_measurement_mobile/static/src/scenario/checkout_package_measurement.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ const new_template =
                     placeholder="Package Height"
                     v-if="state.data.package_requirement.height"
                     v-model="state.data.package.height"
+                    :rules="[() => state.data.package.height > 0 || 'A valid positive number is required']"
```
Comment (simahawk) [inline shopfloor_checkout_package_measurement_mobile/static/src/scenario/checkout_package_measurement.js]: can't we delegate to a method?

Comment (TDu) [reply shopfloor_checkout_package_measurement_mobile/static/src/scenario/checkout_package_measurement.js]: I guess we can. Updated.

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
can you rebase?

**Comment by TDu** [comment]
Rebased :broom:

**Pull request merged**